src/studio/src/designer/backend/Services/Interfaces/IConvertTextsService.cs

```diff
@@ -0,0 +1,16 @@
+namespace Altinn.Studio.Designer.Services.Interfaces
```

Comment:
Extend the ITextsService instead. Primarily to keep functions working on the same thing together.